### PR TITLE
adding some user friendly declarations!

### PR DESCRIPTION
**Pull request by jaybee84**

### CODE REVIEW

**Comment by allaway** [comment]
Just checking, are you still working on this or waiting on review?

**Comment by jaybee84** [comment]
Waiting on review on this one... working on a different branch for the next part of this module

File: R/mod_gene_var.R [hunk 1]
```diff
@@ -25,12 +25,17 @@ mod_gene_variant_ui <- function(id){
       selectizeInput(ns("Genes"), label = "Genes", choices = unique(kairos::jhu_tumor_file@data$Hugo_Symbol),
                      selected = "NF1", multiple = F),
       
+      box(title = "Samples in the selected cohort",
```
Comment (allaway) [inline R/mod_gene_var.R:28]: I don't see this box when I run this locally..not sure why.

Comment (allaway) [reply R/mod_gene_var.R:28]: Nevermind, I was on pr/12 branch instead of pr/15 :)

File: R/mod_gene_var.R [hunk 2]
```diff
@@ -25,12 +25,17 @@ mod_gene_variant_ui <- function(id){
...
+          collapsible = FALSE,
+          textOutput(ns('sample_check'))),
+      
       box(title = "Positional information of variants in tumor samples", 
           status = "primary", solidHeader = TRUE,
           width = 1000,
           collapsible = FALSE,
-          plotOutput(ns('lollipop_plot'))
-      )
+          plotOutput(ns('lollipop_plot')))
```
Comment (allaway) [inline R/mod_gene_var.R:38]: I get this error: "argument "p" is missing, with no default" with the full cohort selected.

**Review by allaway — CHANGES_REQUESTED** [review]
It seems like the lollipop plot no longer works, at least on my end...

File: R/mod_gene_var.R (hunk 2 above)
Comment (jaybee84) [reply R/mod_gene_var.R:38]: hmmm... I am not getting that error message on my end. 

In previous iterations, I got this error when the cohort excluded all samples with genomic var data and the plotting dataset consisted of 0 samples. But in this case the new user messages should take care of this scenario.

**Comment by allaway** [comment]
I can't reproduce this bug on my computer at work....so I guess I was doing something weird, or pulled the wrong branch again. so, LGTM!

**Review by allaway — APPROVED** [review]

File: R/mod_gene_var.R (hunk 2 above)
Comment (allaway) [reply R/mod_gene_var.R:38]: I found the root of this bug: https://github.com/PoisonAlien/maftools/issues/387

Comment (jaybee84) [reply R/mod_gene_var.R:38]: awesome! 

also, I was able to reproduce the bug on my end when I pulled the develop branch today. On the gene_var_1 branch its works well but not on the develop :) possibly due to the issue you mentioned above :)

Comment (jaybee84) [reply R/mod_gene_var.R:38]: Re-opening this conversation since this bug is now bugging me! :(

Comment (jaybee84) [reply R/mod_gene_var.R:38]: To clarify my not-so-clear remark earlier: Acc to issue 387, the issue of maftools layout being overwritten by plotly was fixed in the latest version. But even with the latest version pulled from github, I am encountering this bug when I run the app locally. I reported this observation to PoisonAlien.

**Pull request merged**